package.json

```diff
@@ -32,6 +32,8 @@
     ]
   },
   "dependencies": {
+    "@material-ui/core": "^4.11.1",
```

Comment:
Be sure to mention the install instructions in the README

Reply:
npm install will install all the react libraries, so we should be fine with react libraries.